src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -137,14 +137,14 @@ bool MulticopterLandDetector::_get_freefall_state()
 		return false;
 	}
 
-	if (_ctrl_state.timestamp == 0) {
-		// _ctrl_state is not valid yet, we have to assume we're not falling.
+	if (_sensors.timestamp == 0) {
+		// _sensors is not valid yet, we have to assume we're not falling.
 		return false;
 	}
 
-	float acc_norm = _ctrl_state.x_acc * _ctrl_state.x_acc
-			 + _ctrl_state.y_acc * _ctrl_state.y_acc
-			 + _ctrl_state.z_acc * _ctrl_state.z_acc;
+	float acc_norm = _sensors.accelerometer_m_s2[0] * _sensors.accelerometer_m_s2[0]
+			 + _sensors.accelerometer_m_s2[1] * _sensors.accelerometer_m_s2[1]
+			 + _sensors.accelerometer_m_s2[2] * _sensors.accelerometer_m_s2[2];
```

Comment:
Uncorrected value. Land detector especially needs a bias-corrected value to prevent accel bias from messing with the detection.